client/post-editor/editor-diff-viewer/index.jsx

```diff
@@ -0,0 +1,52 @@
+/**
+ * External dependencies
+ */
+import classNames from 'classnames';
+import { get, map } from 'lodash';
+import React, { PropTypes } from 'react';
+import { connect } from 'react-redux';
+
+/**
+ * Internal dependencies
+ */
+import getPostRevision from 'state/selectors/get-post-revision';
+import getPostRevisionChanges from 'state/selectors/get-post-revision-changes';
+import { normalizeForEditing } from 'state/selectors/utils/revisions';
+
+const EditorDiffViewer = ( { contentChanges, revision } ) => (
+	<div className="editor-diff-viewer">
+		<h1 className="editor-diff-viewer__title">
+			{ get( revision, 'title' ) }
+		</h1>
+		<div className="editor-diff-viewer__content">
+			{ map( contentChanges, ( change, changeIndex ) => {
+				const changeClassNames = classNames( {
+					'editor-diff-viewer__additions': change.added,
+					'editor-diff-viewer__deletions': change.removed,
+				} );
+				return (
+					<span className={ changeClassNames } key={ changeIndex }>
+						{ change.value }
+					</span>
+				);
+			} ) }
+		</div>
+	</div>
+);
+
+EditorDiffViewer.propTypes = {
+	contentChanges: PropTypes.array,
+	postId: PropTypes.number,
+	revision: PropTypes.object,
+	selectedRevisionId: PropTypes.number,
+	siteId: PropTypes.number,
+};
+
+export default connect(
+	( state, ownProps ) => ( {
+		contentChanges: getPostRevisionChanges( state, ownProps.siteId, ownProps.postId, ownProps.selectedRevisionId ),
+		revision: normalizeForEditing(
+			getPostRevision( state, ownProps.siteId, ownProps.postId, ownProps.selectedRevisionId )
+		),
```

Comment:
yep already thought about that, I'm not too sure how to tackle it tbh: I can't use `normalizeForEditing` as a parameter to the selector since a function is an [invalid arg type](https://github.com/Automattic/wp-calypso/blob/master/client/lib/create-selector/index.js#L17). 4 solutions I thought about:

1- Demultyply selectors: `getPostRevisionForEditing`, `getPostRevisionsForEditing`, `getPostRevisionForDisplay` and `getPostRevisionsForDisplay`.
2- Have an internal mapping `string` <-> `function` in the selectors so that we pass a "normalizer" string: `getPostRevision( state, siteId, postId, revisionId, 'editing')` and internally, it will call `normalizeForEditing` (`display` for `normalizeForDisplay`)
3- Do 1- but refactor `getPostRevision` and `getPostRevisions` under the same selector to limit the "API" spread of demultiplying selectors like in 1-? This will make selector usage intent a bit less clear, as sometimes we will do `get(getPostRevisions( state, siteId, postId, revisionId ), 0, null)` using plural and returning an array even though we already know we only want one revision :/ 
4- Find a way to accept (and 'freeze"?) functions as args for the selectors using `createSelector`? <-- I haven't dug much in that direction tbh